django/utils/url_config.py

```diff
@@ -0,0 +1,135 @@
+import inspect
+from collections import defaultdict, namedtuple
+from urllib import parse
+
+from django.core.exceptions import ImproperlyConfigured
+from django.utils import module_loading
+
+BACKENDS = defaultdict(dict)
+
+BaseParsedURL = namedtuple('ParsedURL', 'scheme username password hostname port path options')
+
+
+class ParsedURL(BaseParsedURL):
+    @property
+    def netloc(self):
```

Comment:
Helpful if this should be a public class: A docstring for either `ParsedURL`, `netloc`, or both, explaining what `netloc` does and how `ParsedURL` differs from `urllib.parse.ParseResult`

Reply:
It should definitely have a doc-string, because I've completely forgotten how it differs....!